### DISCUSSION TITLE
Add Installation Option for Frigate 0.15.0 Beta

### DISCUSSION THREAD

**Discussion by alcapone104** [community-scripts/ProxmoxVE#290] [Ideas]
Hello,

I would like to suggest adding an option in the installation script to choose the version of Frigate, specifically to support Frigate 0.15.0 Beta.

Currently, the script installs version 0.14.1 by default. Allowing users to define the desired version (stable or beta) would enhance flexibility and help those who want to test new features.

For instance, a variable like FRIGATE_VERSION could be introduced to enable this functionality:

FRIGATE_VERSION=0.15.0-beta ./install_frigate.sh
This would make it easier to switch between versions without altering the default behavior.

Thank you for considering this improvement, and I appreciate your work on this amazing project!

Best regards,

**Comment by MickLesk** [comment]
Frigate is one of the scripts that has caused the most problems in the past, even with tteck. I don't want to add unstable betas to the script now. 
Or what do the other @community-scripts/contributor say?

**Reply by alcapone104** [reply]
Thank you for your response!

I completely understand your concern about the potential issues caused by Frigate in the past, especially with beta versions being inherently less stable. My suggestion was aimed at providing flexibility for users who are testing or contributing to the development of Frigate.

To address stability concerns, perhaps the option could be implemented as an opt-in feature, clearly marked as "experimental", with a warning about possible issues when using beta versions. This way, the default behavior remains unaffected for most users.

However, I respect the decision and will wait to hear what the other contributors think. Thank you for taking the time to review my suggestion, and I appreciate all the effort you put into maintaining these scripts!

**Reply by judeibe** [reply]
Do we offer beta channels for any script. I am hesitant to start. I consider these scripts as an easy entry into self hosted. If a user desires more custom installs, they have moved beyond the scope of the script and learn more for themselves.

**Reply by alcapone104** [reply]
Thank you for your feedback!

I completely understand your position and your desire to keep these scripts simple and accessible for beginners. It’s a very logical approach, and I respect the goal of maintaining simplicity in your tools.

My intention was simply to suggest an additional option for users who might want to experiment with beta versions while still benefiting from your scripts. However, I understand that this could go beyond the current scope of your tools.

I’ll explore other solutions to customize my installation while continuing to appreciate the advantages your scripts bring to my projects. Thank you again for your work and for taking the time to respond!

**Reply by KurtBelyeu** [reply]
I'm all for it!

**Comment by remz1337** [comment]
I suggest you fork the repo and manually change the Frigate download to a hardcoded url of the beta release. And if you find some breaking changes with v15, don’t hesitate to open a PR 😉

**Comment by Heire** [comment]
Not to hijack this thread but I'm wondering if there has been a look to run frigate on debian 12? now the installation is based on 11.
I would like to test with the rocm integration yet only available on 12. i've been trying to get it running but i'm no expert at the python part.
I don't mind to fork this but is there anywhere some information how these default scripts can be forked to test out and give some feedback perhaps?

**Comment by rottenit** [comment]
I would vote for this as an experimental feature.

**Comment by R3plic8** [comment]
Seems https://ghcr.io/blakeblackshear/frigate:stable points now to v0.15.0-rc2 as of a few days ago.

Would we be able to get an updated script, or even the option to select which version in the helper script?
Seems to be a few fixes with autotracking and openvino so would be very much appreciated.

**Comment by xekil** [comment]
Up, v16 available now, please update script, thanks

**Reply by MickLesk** [reply]
In next 2-4 weeks

**Reply by xekil** [reply]
thank you

**Reply by Hrupek** [reply]
that would be perfect to have v16

**Reply by MickLesk** [reply]
https://github.com/community-scripts/ProxmoxVE/issues/6915#issuecomment-3201055776
